mod_modular_vanilla/hooks/ai/tactical/behavior.nut

```diff
@@ -0,0 +1,10 @@
+::ModularVanilla.QueueBucket.VeryLate.push(function() {
+	::ModularVanilla.MH.hook("scripts/ai/tactical/behavior", function(q) {
+		// MV: Changed
+		// Add skill_container events to allow skills to modify the queried value
+		q.queryTargetValue = @(__original) function( _entity, _target, _skill = null )
```

Comment:
Do you think it is worthwhile to also have the `behavior` as an optional parameter in this function? A skill may want to apply a stronger target value mult for a certain behavior compared to another one. Or is that not necessary?